Functions/QualityControl/xASL_qc_CollectParameters.m

```diff
@@ -96,18 +96,16 @@
 x = xASL_qc_CollectSoftwareVersions(x);
 
 % Module-specific software versions:
-switch ScanType
-    case 'Structural'
-        x.Output.(ScanType).Version_CAT12 = x.Output.SoftwareVersion.CAT12;
-        x.Output.(ScanType).Version_LST = x.Output.SoftwareVersion.LST;
-        
-        % General software versions (put this in structural only, to avoid redundant output
-        x.Output.(ScanType).Version_ExploreASL = x.Output.SoftwareVersion.ExploreASL;
-        x.Output.(ScanType).Version_Matlab = x.Output.SoftwareVersion.Matlab;
-        x.Output.(ScanType).Version_SPM12 = x.Output.SoftwareVersion.SPM12;        
-    case {'dwi', 'func', 'ASL'}
-        x.Output.(ScanType).(x.SESSIONS{iSession}).Version_FSL = x.Output.SoftwareVersion.FSL;
-end
+x.Output.(ScanType).Version_CAT12 = x.Output.SoftwareVersion.CAT12; % CAT12
```

Comment:
I agree to print ExploreASL version in all modules. 
But it doesn't IMHO make sense to print CAT12 and LST for other than structural modules.

Reply:
Done in 128171247d5b8d8d8a4d73bc7cc72c72846cbc3a